docs/.vuepress/config.js

```diff
@@ -57,6 +59,14 @@ module.exports = {
     editLinks: true,
     contributors: false,
   }),
+  async onPrepared(app) {
+    const componentsApi = await generateComponentsApi();
+
+    await app.writeTemp(
+      'componentsApi.js',
+      `export const apiData = ${JSON.stringify(componentsApi)}`
+    );
```

Comment:
The way it works is by generating the docs in the Node api when the app is created, then we save the data in a temp file to pass it to the client side. In the client side we take the data that is relevant to our component, and parse it into tables